proto/sifnode/oracle/v1/types.proto

```diff
@@ -59,4 +60,12 @@ message CrossChainFeeConfig {
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"minimum_burn_cost\""
   ];
+}
+
+// ProphecySignatures include the ethereum addressed and signatures.
+message ProphecySignatures {
+  sifnode.oracle.v1.NetworkDescriptor network_descriptor = 1
+      [ (gogoproto.moretags) = "yaml:\"network_descriptor\"" ];
+  repeated string ethereum_address = 2;
+  repeated string signatures = 3;
```

Comment:
Should be named to ProphecySignature and signature, not signatures as each message relayed to sifchain will only contain the signature that witness has signed and not all of the signatures.

Reply:
Is signatures being used to define the data structure that all of the signatures for a single prophecy will be stored in?

Reply:
this data structure is not for message. keeper use it to store all signatures and other information about this prophecy.

Reply:
Ok. Got it.